specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/providers.json

```diff
@@ -152,6 +159,81 @@
           "description": "Blob duration of specification."
         }
       }
+    },
+    "MetricSpecification": {
+      "description": "Metric specification of operation.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of metric specification."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of Metric specification."
+        },
+        "displayDescription": {
+          "type": "string",
+          "description": "Display description of Metric specification."
+        },
+        "unit": {
+          "type": "string",
+          "description": "The metric unit. Possible values include: 'Bytes', 'Count', 'Milliseconds'"
```

Comment:
Should this and other constrained properties be enums to better support SDKs and validation by the CLI?

Reply:
Or is this output only?

Reply:
Yes, this one is only used in 200 response.